pkg/controllers/vdb/restart_reconciler.go

```diff
@@ -371,6 +377,20 @@ func (r *RestartReconciler) reipNodes(ctx context.Context, pods []*PodFact) (ctr
 	opts := []reip.Option{
 		reip.WithInitiator(r.InitiatorPod, r.InitiatorPodIP),
 	}
+	// If a communal path is set, include all of the EON parameters.
+	if r.Vdb.Spec.Communal.Path != "" {
+		// build communal storage params if there is not one
+		if r.ConfigurationParams == nil {
+			res, err := r.ConstructConfigParms(ctx)
+			if err != nil {
```

Comment:
We need to check res too. There is a helper for that.
```suggestion
			if verrors.IsReconcileAborted(res, err) {
```

Reply:
The res has been checked in lines 76 and 82 of ConstructConfigParms(). Are those checks not enough?